packages/react-wildcat-handoff/src/utils/serverRender.js

```diff
@@ -37,6 +37,8 @@ module.exports = function serverRender(cfg) {
             } else {
                 let initialData = null;
 
+                const has404Status = has404StatusRouterProperty(renderProps.components);
```

Comment:
So this is too strict. This check would only ever match on 404. Instead you should check that `routerProps.status` is defined, and if so use that value. This will allow you to not only supply 404s but 500s, 501s, etc.